Product/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+class Main_category(models.Model):
```

Comment:
클래스 이름에서는 띄어쓰기는 언더바를 사용하지 않고 대문자로 시작하는 카멜케이스(camelCase)를 사용합니다
```suggestion
class MainCategory(models.Model):
```

아래 서브 카테고리, 프로덕트 그룹 모두 통일해주세요